Gemfile

```diff
@@ -36,12 +36,17 @@ gem "bootsnap", require: false
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 # gem "rack-cors"
 
+gem 'devise_token_auth', '~> 1.2.1'
+
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[ mri mingw x64_mingw ]
   gem "rspec", "~> 3.11.0"
   gem "rspec-rails", "~> 6.0.0"
+  gem "factory_bot_rails", "~> 6.2.0"
+  gem "faker", git: 'https://github.com/faker-ruby/faker.git', branch: 'main'
```

Comment:
If you are not having the `uninitialized constant Faker::[some_class]` error, I prefer to use the last released version of this gem. One tip is to add it without versions, and then look into the Gemfile.lock for the specific version installed and then add the version control in the gemfile